src/schema/v2/artist/index.ts

```diff
@@ -220,55 +220,46 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
               "When true, will only return records for allowed artists.",
           },
         }),
-        resolve: ({ _id }, options, { auctionLotLoader }) => {
+        resolve: ({ _id }, options, _context) => {
           if (options.recordsTrusted && !includes(auctionRecordsTrusted, _id)) {
             return null
           }
 
           // Convert `after` cursors to page params
-          const {
```

Comment:
(🐱 can add a magic comment for the linter, `// eslint-disable-next-line` kind of thing vs. having to remove or comment out the code, fyi)